cli/src/main/java/bisq/cli/app/BisqCliMain.java

```diff
@@ -41,60 +45,28 @@
 
     private final ManagedChannel channel;
     private final CliCommand cmd;
+    private final OptionParser cmdParser;
 
     public static void main(String[] args) {
-        new BisqCliMain("localhost", 8888);
+        new BisqCliMain("localhost", 8888, args);
     }
 
-    private BisqCliMain(String host, int port) {
+    private BisqCliMain(String host, int port, String[] params) {
```

Comment:
```suggestion
    private BisqCliMain(String host, int port, String[] args) {
```

For consistency and conventionality.